backend/src/data_source/user.rs

```diff
@@ -37,6 +37,13 @@ struct PgUser {
     id: Uuid,
     first_name: String,
     last_name: String,
+    phone: String,
+    student_number: Option<i64>,
+    nkbv_number: Option<i64>,
+    sportcard_number: Option<i64>,
+    ice_contact_name: String,
+    ice_contact_email: String,
+    ice_contact_phone: String,
```

Comment:
Same comment as for the SQL file: Would be nice to make them mandatory, but likely not possible.